src/App/App.tsx

```diff
@@ -1,89 +1,98 @@
-import React, { ReactElement, useEffect, useState } from 'react';
-import { Route, Switch } from 'react-router-dom';
-import Login from '../Login/Login';
-import Header from '../Header/Header';
-import Dashboard from '../Dashboard/Dashboard';
-import About from '../About/About';
-import AllCocktailsPage from '../AllCocktailsPage/AllCocktailsPage';
-import MyCocktails from '../MyCocktails/MyCocktails';
-import CocktailDetails from '../CocktailDetails/CocktailDetails';
+import React, { ReactElement, useEffect, useState } from "react";
+import { Route, Switch } from "react-router-dom";
+import Login from "../Login/Login";
+import Header from "../Header/Header";
+import Dashboard from "../Dashboard/Dashboard";
+import About from "../About/About";
+import AllCocktailsPage from "../AllCocktailsPage/AllCocktailsPage";
+import MyCocktails from "../MyCocktails/MyCocktails";
+import CocktailDetails from "../CocktailDetails/CocktailDetails";
 import { getAllCocktails } from "../apiCalls";
-import './App.scss';
+import "./App.scss";
 
 export interface AllCocktailsDetails {
-		strDrink: string,
-		strDrinkThumb: string,
-		idDrink: string
+  strDrink: string;
+  strDrinkThumb: string;
+  idDrink: string;
 }
 
 const App: React.SFC = () => {
-	const [ username, setUsername ] = useState('');
-	const [ loggedIn, setLoggedIn ] = useState(false);
-	const [ allCocktails, setAllCocktails ] = useState<AllCocktailsDetails[]>([{strDrink: '',
-		strDrinkThumb: '',
-		idDrink: ''}]);
-  const [error, setError] = useState('');
+  const [username, setUsername] = useState("");
+  const [loggedIn, setLoggedIn] = useState(false);
+  const [allCocktails, setAllCocktails] = useState<AllCocktailsDetails[]>([
+    { strDrink: "", strDrinkThumb: "", idDrink: "" },
+  ]);
+  const [favCocktails, setFavCocktails] = useState<string[]>([]);
+  const [error, setError] = useState("");
 
-	//store favorite in app
-	//fn that will filter searched input
+  //fn that will filter searched input
 
-	const fetchAllCocktails = async (): Promise<any> => {
+  // API Calls
+  const fetchAllCocktails = async (): Promise<any> => {
     try {
       const data: AllCocktailsDetails[] = await getAllCocktails();
       return setAllCocktails(data);
     } catch (error) {
       setError(error.message);
     }
-	};
+  };
 
-	useEffect(() => {fetchAllCocktails()}, []);
+  useEffect(() => {
+    fetchAllCocktails();
+  }, []);
 
-	//add to favorites (id);
-	// .push ()
+  // Functions
+  
+  const toggleFavorites = (drinkID: string): any => {
```

Comment:
@asodemann18 & I created functionality to add and remove an id that toggles into the favorites array.